src/vl-select.js

```diff
@@ -61,6 +61,24 @@ export class VlSelect extends vlFormValidationElement(nativeVlElement(HTMLSelect
     }
   }
 
+  _dataRequiredChangedCallback(oldValue, newValue) {
+    if (newValue && !oldValue) {
+      this.setAttribute('required', '');
+    }
+    if (!newValue && oldValue) {
+      this.removeAttribute('required');
+    }
+  }
+
+  _requiredChangedCallback(oldValue, newValue) {
```

Comment:
ik denk dat deze _dataRequired... moet noemen en de bovenstaande _required als ik naar de inhoud van de methodes kijk

Reply:
Nee als required changed moet ge data-required daarmee in sync houden en omgekeerd. Dus lijkt me correct.